### PR TITLE
[Term Entry] Python built-in functions slice() #5117

### LINKED ISSUE #5117: [Term Entry] Python built-in functions slice()
### Reason/inspiration (optional)

We would like a new term entry on the `slice()` built-in function in Python. The entry should go in a new file under `docs/content/python/concepts/built-in-functions/terms/slice/slice.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by aSagCoder**
### Description

- added the term entry for in-built Python function .slice(). 
- Can be located here - content/python/concepts/built-in-functions/terms/slice/slice.md

### Issue Solved

Closes #5117 

### Type of Change

- Adding a new entry
 - ~~Editing an existing entry (fixing a typo, bug, issues, etc)~~
- ~~Updating the documentation~~

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/slice/slice.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```pseudo
+slice(start, stop, step)
+```
+
+where it takes three parameters:
+- `start:` This integer indicates where the slicing of the object begins and is optional. Default value is None if not provided. 
+- `stop:` This integer indicates up to where the slicing takes place. It stops at index stop-1 (the last element of the sequence). 
+- `step:` This integer value is used to fix the increment between each index for slicing and is optional. Its default value is null if not provided. 
+
```
Comment (Sriparno08) [inline content/python/concepts/built-in-functions/terms/slice/slice.md:26]: Modifications:

- Let's remove the introductory line as it's not needed
- Let's do some minor changes in the bulleted list

```suggestion
- `start`: This optional integer indicates where the slicing of the object begins. The default value is `None` if not provided.
- `stop`: This integer indicates up to where the slicing takes place. It stops at index `stop-1` (the last element of the sequence).
- `step`: This optional integer fixes the increment between each index for slicing. The default value is `None` if not provided.
```

File: content/python/concepts/built-in-functions/terms/slice/slice.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Example
+
+The following example shows the use of the `.slice()' function:
```
Comment (Sriparno08) [inline content/python/concepts/built-in-functions/terms/slice/slice.md:30]: Let's do some minor changes:

```suggestion
The following example shows the use of the `slice()` function:
```

File: content/python/concepts/built-in-functions/terms/slice/slice.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+```py
+#create a tuple with some values in it
+fruits = ("Apple", "Banana", "Orange", "Kiwi", "Strawberry", "Plum", "Watermelon")
+
+#create a slice object
+a = slice(3)
+
+#return the slice result
+print(fruits[a])
+
```
Comment (Sriparno08) [inline content/python/concepts/built-in-functions/terms/slice/slice.md:41]: Let's start the comments using capital letters and do some minor changes:

```suggestion
# Create a tuple with some values in it
fruits = ("Apple", "Banana", "Orange", "Kiwi", "Strawberry", "Plum", "Watermelon")

# Create a slice object
a = slice(3)

# Return the resultant slice object
print(fruits[a])
```

File: content/python/concepts/built-in-functions/terms/slice/slice.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+```
+The output of the code is as follows:
+
+```shell
+('Apple', 'Banana', 'Orange')
+
+```
```
Comment (Sriparno08) [inline content/python/concepts/built-in-functions/terms/slice/slice.md:48]: Let's do some minor changes:

````suggestion
```

The output of the code is as follows:

```shell
('Apple', 'Banana', 'Orange')
```
````

File: content/python/concepts/built-in-functions/terms/slice/slice.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+#string slicing
+my_string = 'CompetetiveProgramming'
+slice_obj1 = slice(5)
+slice_obj2 = slice(1, 8, 2)
+
+print("Resulting slices:")
+print(my_string[slice_obj1])
+print(my_string[slice_obj2])
+
+```
```
Comment (Sriparno08) [inline content/python/concepts/built-in-functions/terms/slice/slice.md:66]: Modifications:

- Let's remove 'if applicable' and the comments as they're not needed
- Let's do some minor changes

````suggestion
## Codebyte Example

The following codebyte example demonstrates how the `slice()` function is used to create a slice object from a given sequence (here, a string) between a defined range:

```codebyte/python
my_string = 'CompetitiveProgramming'
slice_obj1 = slice(5)
slice_obj2 = slice(1, 8, 2)

print("Resulting slices:")
print(my_string[slice_obj1])
print(my_string[slice_obj2])
```
````

**Review by Sriparno08 — CHANGES_REQUESTED** [review]
Hey @aSagCoder, thanks for contributing to Codecademy Docs!

I've listed all the modifications that you need to make in your entry below:

**Comment by aSagCoder** [comment]
> Hey @aSagCoder, thanks for contributing to Codecademy Docs!
> 
> I've listed all the modifications that you need to make in your entry below:

Hi there, thanks for the suggestions! I have gone over those and committed them.

**Review by Sriparno08 — APPROVED** [review]
Good job, @aSagCoder!

This entry looks good for a second round of review.

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @aSagCoder 😄 

The entry is ready to be merged! 🚀

**Pull request merged**